okd/src/microshift-okd-multi-build.Containerfile

```diff
@@ -52,4 +53,8 @@ RUN ${OKD_CONFIG_SCRIPT} && rm -rf ${OKD_CONFIG_SCRIPT}
 # Create a systemd unit to recursively make the root filesystem subtree
 # shared as required by OVN images
 COPY ./packaging/imagemode/systemd/microshift-make-rshared.service /etc/systemd/system/microshift-make-rshared.service
-RUN systemctl enable microshift-make-rshared.service
\ No newline at end of file
+RUN systemctl enable microshift-make-rshared.service
+# In case of flannel we don't need openvswitch service which is by default enabled as part
+# once microshift is installed so better to disable it because it cause issue when required
+# module is not enabled.
+RUN if [ "$WITH_FLANNEL" -eq 1 ]; then systemctl disable openvswitch; fi
```

Comment:
Nit: can we move this to line 45 to avoid additional layer? 
The comment can go under 'Installing MicroShift and cleanup"

Reply:
@ggiguash Done